pkg/users/authorized-keys-panel.js

```diff
@@ -85,79 +92,94 @@
+                id="account-authorized-keys-list"
+                showHeader={false}
+                emptyCaption={error}
+                columns={ [
+                    { title: _("Name"), header: true },
+                    { title: _("Fingerprint") },
+                    { title: "" },
+                ] }
+                rows={keys.map(k => {
+                    return {
+                        columns: [
+                            { title: k.comment || _("Unnamed"), props: { width: 20 } },
+                            { title: k.fp || _("Invalid key"), props: { width: 80 } },
+                            {
+                                title: <OverflowMenu breakpoint="lg">
+                                    <OverflowMenuContent>
+                                        <OverflowMenuGroup groupType="button">
+                                            <OverflowMenuItem>
+                                                <Button key={k.fp} variant="secondary" onClick={() => remove_key(k.raw)}>
+                                                    {_("Remove")}
+                                                </Button>
+                                            </OverflowMenuItem>
+                                        </OverflowMenuGroup>
+                                    </OverflowMenuContent>
+                                    <OverflowMenuControl>
+                                        <Dropdown position="right"
+                                                  onSelect={() => {
+                                                      if (openedMenu.indexOf(k.fp) >= 0)
+                                                          setOpenedMenu(openedMenu.filter(m => m !== k.fp));
+                                                      else
+                                                          setOpenedMenu([...openedMenu, k.fp]);
+                                                  }}
+                                                  toggle={
+                                                      <KebabToggle
+                                                      onToggle={open => {
+                                                          if (open)
+                                                              setOpenedMenu([...openedMenu, k.fp]);
+                                                          else
+                                                              setOpenedMenu(openedMenu.filter(m => m !== k.fp));
+                                                      }}
+                                                      />
+                                                  }
+                                                  isOpen={openedMenu.indexOf(k.fp) >= 0}
+                                                  isPlain
+                                                  dropdownItems={[<OverflowMenuDropdownItem key="delete" isShared onClick={() => remove_key(k.raw)}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18014-20221213-102921-2058c207-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)